mindmeld/components/dialogue.py

```diff
@@ -717,12 +717,13 @@ class AutoEntityFilling:
     _logger = mod_logger.getChild("AutoEntityFilling")
     """Class logger."""
 
-    def __init__(self, entrance_handler, form, app):
+    def __init__(self, handler, form, app, clear_prev=False):
         self._app = app
-        self._entrance_handler = entrance_handler
+        self._entrance_handler = handler
         self._form = form
         self._local_form = None
         self._prompt_turn = None
+        self._clear_prev = clear_prev
```

Comment:
Let's call it `clear_previous` for clarity.

Reply:
Removed this.